Dockerfile

```diff
@@ -0,0 +1,39 @@
+# Base on offical Node.js Alpine image
+FROM node:current-alpine AS base
+
+# Set working directory
+WORKDIR /base
+
+# Copy package.json and package-lock.json before other files
+# Utilise Docker cache to save re-installing dependencies if unchanged
+COPY package*.json yarn.lock ./ 
+
+# Install dependencies
+RUN yarn install --frozen-lockfile
```

Comment:
Quick question, should we also pass `--production` too?

Reply:
Quick question, since this is the part where we install dependencies, should we also add `ENV NODE_ENV=production` in this stage?

Reply:
Yes, we should do so in every stage. I will put it in.

Reply:
About `NODE_ENV=production`, because this (build) step only relates to a production build, so we're explicitly setting the `NODE_ENV` to production.